protected/models/forms/createUpdateProject.json

```diff
@@ -10,18 +10,27 @@
           "title": "Project name",
           "isRequired": true
         },
+        {
+          "type": "currentlanguage",
+          "name": "lang",
+          "visible": false,
+          "isRequired": true,
+          "readOnly": true
+        },
         {
           "type": "checkbox",
           "name": "languages",
           "title": "Project languages",
           "description": "These will be available for workspace & facility data",
+          "defaultValue": [
+            "en"
+          ],
           "isRequired": true,
           "choicesByUrl": {
-            "url": "/api-proxy/core/configuration/locales",
+            "url": "/api-proxy/core/configuration/locales?_lang={lang}",
```

Comment:
This will not work in json, right?

Reply:
actually it does, cause the language is sent from the settings :) trough an hidden input field

Reply:
used the elements and moved the group to the javascript side.